src/denish/Assignment23.java

```diff
@@ -0,0 +1,27 @@
+package denish;
+
+public class Assignment23 {
+	public static void main(String[] args) {
+		
+		String str ="TecHNo_cR6ediTs";
+		int ucount=0;
+		int lcount=0;
+		int dcount=0;
+		int specialcount=0;
+		
+		for(int index=0; index<str.length(); index++) {
+			if(Character.isUpperCase(str.charAt(index))) {
```

Comment:
It would be better if you create char ch = str.charAt(index) outside of condition, so you no need to write it again and again, you have to just write ch